x/bank/types/expected_keepers.go

```diff
@@ -35,5 +35,6 @@ type AccountKeeper interface {
 
 // BankHooks event hooks for bank sends
 type BankHooks interface {
-	BeforeSend(ctx sdk.Context, from sdk.AccAddress, to sdk.AccAddress, amount sdk.Coins) error // Must be before any send is executed
+	TrackBeforeSend(ctx sdk.Context, from, to sdk.AccAddress, amount sdk.Coins)       // Must be before any send is executed
+	BlockBeforeSend(ctx sdk.Context, from, to sdk.AccAddress, amount sdk.Coins) error // Must be before any send is executed
```

Comment:
Nit: since this is an interface, let's be sure to godoc them -- it's not clear to me what they do. No need to godoc the actual method implementation(s).